system/include/emscripten/val.h

```diff
@@ -109,6 +113,9 @@ bool _emval_delete(EM_VAL object, EM_VAL property);
 EM_VAL _emval_await(EM_VAL promise);
 EM_VAL _emval_iter_begin(EM_VAL iterable);
 EM_VAL _emval_iter_next(EM_VAL iterator);
+// Can't wrap these into `#if _LIBCPP_STD_VER >= 20` because that breaks siginfo checks.
```

Comment:
Should we build siginfo with c++20?

Reply:
Ah would that help / can we do that? If so, maybe! I never looked into how these run.

Reply:
Seems pretty straightforward... I'll try that now.